system-test/src/main/java/org/apache/mesos/elasticsearch/systemtest/Main.java

```diff
@@ -56,12 +59,12 @@ private static void seedData(MesosCluster cluster, ElasticsearchSchedulerContain
         String taskHttpAddress;
         try {
             ESTasks esTasks = new ESTasks(TEST_CONFIG, schedulerContainer.getIpAddress());
-            new TasksResponse(esTasks, cluster.getConfig().getNumberOfSlaves(), "TASK_RUNNING");
+            new TasksResponse(esTasks, 3, "TASK_RUNNING");
```

Comment:
3 will be difficult to understand. Should be replaced by cluster.getSlaves().length